docs/lib/datastore/fragments/android/datastore-events.md

```diff
@@ -0,0 +1,48 @@
+For instance, to listen to see if the network status is active, you could set up the following listener:
+
+```js
+// Create listener
+const listener = Hub.listen('datastore', async hubData => {
+  const  { event, data } = hubData.payload;
+  if (event === 'networkStatus') {
+    console.log(`User has a network connection? ${data.active}`)
```

Comment:
```suggestion
    console.log(`User has a network connection ${data.active}`)
```

Reply:
Or why the the `?`

Reply:
Not sure. It was on the JS example that I copied, I believe. I guess the idea is that it would read "...connection? true" or "connection? false" -- not great.

Reply:
Changed it to a colon.